socs/agents/hwp_pid/agent.py

```diff
@@ -339,49 +279,43 @@ def make_parser(parser=None):
         parser = argparse.ArgumentParser()
 
     # Add options specific to this agent
-    pgroup = parser.add_argument_group('Agent Options')
-    pgroup.add_argument('--ip')
-    pgroup.add_argument('--port')
-    pgroup.add_argument('--verbose', '-v', action='count', default=0,
-                        help='PID Controller verbosity level.')
-    pgroup.add_argument('--mode', type=str, default='acq',
-                        choices=['init', 'acq'],
-                        help="Starting operation for the Agent.")
+    pgroup = parser.add_argument_group("Agent Options")
+    pgroup.add_argument("--ip")
+    pgroup.add_argument("--port")
+    pgroup.add_argument(
+        "--verbose",
+        "-v",
+        action="count",
+        default=0,
+        help="PID Controller verbosity level.",
+    )
+    pgroup.add_argument("--mode")
     return parser
 
 
 def main(args=None):
     parser = make_parser()
-    args = site_config.parse_args(agent_class='HWPPIDAgent',
-                                  parser=parser,
-                                  args=args)
-
-    init_params = False
-    if args.mode == 'init':
-        init_params = {'auto_acquire': False}
-    elif args.mode == 'acq':
-        init_params = {'auto_acquire': True}
+    args = site_config.parse_args(agent_class="HWPPIDAgent", parser=parser, args=args)
 
     agent, runner = ocs_agent.init_site_agent(args)
-    hwppid_agent = HWPPIDAgent(agent, ip=args.ip,
-                               port=args.port,
-                               verbosity=args.verbose)
-    agent.register_task('init_connection', hwppid_agent.init_connection,
-                        startup=init_params)
-    agent.register_process('acq', hwppid_agent.acq,
-                           hwppid_agent._stop_acq)
-    agent.register_task('tune_stop', hwppid_agent.tune_stop)
-    agent.register_task('tune_freq', hwppid_agent.tune_freq)
-    agent.register_task('declare_freq', hwppid_agent.declare_freq)
-    agent.register_task('set_pid', hwppid_agent.set_pid)
-    agent.register_task('get_freq', hwppid_agent.get_freq)
-    agent.register_task('get_target', hwppid_agent.get_target)
-    agent.register_task('get_direction', hwppid_agent.get_direction)
-    agent.register_task('set_direction', hwppid_agent.set_direction)
-    agent.register_task('set_scale', hwppid_agent.set_scale)
 
+    if args.mode is not None:
+        agent.log.warn("--mode agrument is deprecated.")
```

Comment:
Since this is true, we should remove it from the example config block in the docs.